doc/source/user_guide/visualization.rst

```diff
@@ -0,0 +1,37 @@
+Data visualization
+------------------
+
+Data visualization takes an important part in image processing. Data can be
+a simple unique 2D image or a more complex with multidimensional aspects: 3D
+in space, timeslapse, multiple channels.
+
+Therefore, the visualization strategy will depend on the data complexity and
+a range of tools external to scikit-image can be used for this purpose.
+Historically, scikit-image provided viewer tools but new powerful softwares
+are now developed and must be preferred.
```

Comment:
```suggestion
are now available and must be preferred.
```

Reply:
some of these packages have been around the block for more than a decade so I would not insist too much on the new/developed :-).

Reply:
haha You're right :) I had only napari in my buffer.